test/Generics/inheritance.swift

```diff
@@ -55,7 +55,7 @@ func testGenericInherit() {
 }
 
 
-struct SS<T> : T { } // expected-error{{inheritance from non-protocol type 'T'}}
+struct SS<T> : T { } // expected-error{{class can only inherit from protocol in extension declaration 'T'}}
```

Comment:
This is incorrect because declaration is a `struct` but diagnostic says is a `class`.

Reply:
Can you have a look now?